docs/02-app/01-building-your-application/05-optimizing/05-static-assets.mdx

```diff
@@ -7,18 +7,21 @@ Next.js can serve static files, like images, under a folder called `public` in t
 
 For example, if you add `me.png` inside `public`, the following code will access the image:
 
-```jsx
+```jsx title="Avatar.js"
 import Image from 'next/image'
 
-function Avatar() {
+export function Avatar() {
   return <Image src="/me.png" alt="me" width="64" height="64" />
 }
-
-export default Avatar
 ```
 
-This folder is also useful for `robots.txt`, `favicon.ico`, Google Site Verification, and any other static files (including `.html`)!
+<PagesOnly>
+
+This folder is also useful for `robots.txt`, `favicon.ico`, Google Site Verification, and any other static files (including `.html`). But make sure to not have a static file with the same name as a file in the `pages/` directory, as this will result in an error. [Read more](/docs/messages/conflicting-public-file-page).
+
+</PagesOnly>
```

Comment:
Should we add a `<AppOnly>` section here too that mentions the file based / metadata api so people don't put files like `robots.txt`, `favicon.ico`, etc. here?

Reply:
Good suggestion